csrankings-a.csv

```diff
@@ -1652,6 +1653,7 @@ Annabelle McIver,Macquarie University,https://researchers.mq.edu.au/en/persons/a
 Annalisa De Bonis,University of Salerno,http://www.di.unisa.it/professori/debonis,XpwdqF8AAAAJ
 Annalisa Massini,Sapienza University of Rome,https://twiki.di.uniroma1.it/twiki/view/Users/AnnalisaMassini,u2H7ZqsAAAAJ
 Annalu Waller,University of Dundee,https://www.computing.dundee.ac.uk/about/staff/27,x_YEVQoAAAAJ
+Annamaria Mesaros,Tampere University,https://www.tuni.fi/en/annamaria-mesaros,tOvdEZIAAAAJ

```

Comment:
Research fellow, not a tenure-track professor - please remove.

Reply:
Academy research fellow and tenure-track professor are not mutually exclusive in Finland. This is very common, because the former has funding that comes directly from the Academy of Finland, while the latter does not necessarily have that, rather directly from the university.

How about we replace with this URL instead?

https://researchportal.tuni.fi/en/persons/annamaria-mesaros

Reply:
That's fine, please do that.

Reply:
Addressed via commit  7cf758c94e